pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/CmdFunctionsTest.java

```diff
@@ -831,7 +845,7 @@ public void testUpdateAuthData() throws Exception {
                 "--jar", URL,
                 "--tenant", "sample",
                 "--namespace", "ns1",
-                "--className", DummyFunction.class.getName(),
+                "--classname", DummyFunction.class.getName(),
```

Comment:
Why are we changing these tests?
Are we breaking compatibility?
I would prefer to not change existing tests if not needed

Reply:
Both `className` and `classname` are accepted. `className` is the legacy deprecated param and only accepted for backward compat. Nowadays people should use `classname` so I think it's better to test with what people really use. WDYT ?

Reply:
Do we have a test to ensure  backward compatibility?

Reply:
There are a lot of deprecated params in CmdFunctions and no tests for compatibility. Let's deal with this in another PR. I removed the param name change. PTAL.